docs/tutorial/troubleshooting/migrate_pool_by_migrating_disks.md

```diff
@@ -0,0 +1,314 @@
+```sh
+$ kubectl apply -f cspc.yaml
+```
+
+### Step4: Update the cspi spec nodeSelectors, labels and NodeName
+
+This step is required to update the CSPI with correct node details by looking at the node where corresponding CSPI blockdevices were attached.
+```sh
+apiVersion: cstor.openebs.io/v1
+kind: CStorPoolInstance
+metadata:
+  name: cstor-cspc-chjg
+  namespace: openebs
+  labels:
+    kubernetes.io/hostname: ip-192-168-95-129
+  ...
+  ...
+spec:
+  dataRaidGroups:
+  - blockDevices:
+    - blockDeviceName: blockdevice-c783e51a80bc51065402e5473c52d185
+  hostName: ip-192-168-95-129
+  nodeSelector:
+    kubernetes.io/hostname: ip-192-168-95-129
+  poolConfig:
+     ...
+     ...
+```
+Get the node details on which the **blockdevice-c783e51a80bc51065402e5473c52d185** was attached and fter fetching node details update hostName, nodeSelector values and kubernetes.io/hostname values in lables of CSPI with new details.
+```sh
+apiVersion: cstor.openebs.io/v1
+kind: CStorPoolInstance
+metadata:
+  name: cstor-cspc-chjg
+  namespace: openebs
+ labels:
+    kubernetes.io/hostname: ip-192-168-75-156
+  ...
+  ...
+spec:
+  dataRaidGroups:
+  - blockDevices:
+    - blockDeviceName: blockdevice-c783e51a80bc51065402e5473c52d185
+  hostName: ip-192-168-75-156
+  nodeSelector:
+    kubernetes.io/hostname: ip-192-168-75-156
+  poolConfig:
+     ...
+     ...
+```
+Update using `kubectl edit cspi <cspi_name> -n openebs`
+
+**NOTE**: Repeat the same process for all other CSPIs which are in pending state and belongs to update CSPC.
+
+### Step5: Verification
+
+This step is identify whether pools has been imported or not. Updated CSPI will get events saying pool is successfully imported.
+```sh
+$ kubectl describe cspi cstor-cspc-xs4b -n openebs
+...
+...
+Events:
+  Type    Reason         Age    From               Message
+  ----    ------         ----   ----               -------
+  Normal  Pool Imported  2m48s  CStorPoolInstance  Pool Import successful: cstor-07c4bfd1-aa1a-4346-8c38-f81d33070ab7
+```
+
+### Step6: Scaleup the cStor admission server and update the validatingwebhookconfiguration
+
+This step is required to bring back the cStor admission server into Running state. As well as admssion server is required to validate the modifications made to CSPC API in future.
+```sh
+$ kubectl scale deploy openebs-cstor-admission-server -n openebs --replicas=1
+deployment.extensions/openebs-cstor-admission-server scaled
+```
+
+Update **failurePolicy to Fail** in validatingwebhookconfiguration.
+```sh
+$ kubectl edit validatingwebhookconfiguration openebs-cstor-validation-webhook
+validatingwebhookconfiguration.admissionregistration.k8s.io/openebs-cstor-validation-webhook edited
+```
+
+## NOTE: In upcoming release we need to perform only [STEP3](#step3-update-the-cspc-spec-nodeselector) (No need of performing other steps).
```

Comment:
We can remove this can track it in an issue to update the doc once the release is out. 
Also, why performing only step3 will work? Any changes we are making in this release in this context?

Reply:
>  why performing only step3 will work? Any changes we are making in this release in this context?

In the future, we need to make this kind of changes but it might not be this release.